dio/test/cancel_token_test.dart

```diff
@@ -19,5 +19,27 @@ void main() {
     test('cancel without use does not throw (#1765)', () async {
       CancelToken().cancel();
     });
+
+    test('cancels multiple requests', () async {
```

Comment:
These tests didn't seem covered clients closed status?

Reply:
No they didn't. And I am currently also not testing if `abort()` is called - probably need a mock `HttpClient` and mocked requests for this.

Reply:
I updated the tests to use mocks and verify that `abort` is actually being called.